### ISSUE TITLE
Chore: Set up build

### ISSUE THREAD

**Issue by pahimar** [pahimar/Equivalent-Exchange#49]
Set up building of mod on Travis
- [ ] produces deobf jar
- [ ] signs jar
- [ ] publishes to Maven (TBD which one)
- [ ] publishes to CurseForge on tag

**Comment by JLLeitschuh** [comment]
If you are rewriting your build from scratch, you should look into writing your build logic with the new [Gradle Kotlin DSL](https://github.com/gradle/kotlin-dsl). More than happy to help you out with it if you are interested.

**Comment by pahimar** [comment]
Thank you for the suggestion :-) This is using a derivative of the provided ForgeGradle buildscript and I want to keep it similar so that other modders (beginning and experienced) have a point of reference.

**Comment by JLLeitschuh** [comment]
Fair enough. Maybe I'll work with the Forge Team to provide an example that uses the Gradle Kotlin DSL.

Best of luck @pahimar. I've enjoyed your videos with Direwolf.
